sites/feedback/feedback-docs-en.html

```diff
@@ -0,0 +1,245 @@
+
+
+<h2>Working example</h2>
+<ul>
+	<li><a href="feedback-en.html">Default page feedback form example</a></li>
+	<li><a href="/sites/page-details/page-details-v1-en.html">Default page feedback form version 1 example</a></li>
+</ul>
+
+<h2>How to implement</h2>
+<ul>
+	<li><p>To show the page feedback form, create a <code>&lt;div&gt;</code> with the following data attribute: <code>data-ajax-replace="/sites/feedback/ajax/report-problem-en.html"</code>.</p>
+		<p>For the basic HTML version, a link element should be created inside a <code>&lt;div&gt;</code> that points to the report a problem form page: <code>&lt;div&gt;&lt;a class="btn btn-default btn-block" href="https://www.canada.ca/en/report-problem.html"&gt;Report a problem on this page&lt;/a&gt;&lt;/div&gt;</code></p>
+		<p>For more information about Data Ajax plugin, please visit <a href="https://wet-boew.github.io/wet-boew/docs/ref/data-ajax/data-ajax-en.html">Data Ajax documentation page</a></p>
+	</li>
+</ul>
+
+<h2>Evaluation and report</h2>
+<p>There is no evaluation and report available for this component.</p>
+
+<h2>API (Version 2.0)</h2>
+
+<table class="table table-bordered">
+	<tr>
+		<th>CSS Class</th>
+		<th>Template</th>
+		<th>Visual rendering</th>
+		<th>Schema</th>
+	</tr>
+	<tr>
+		<td>n.a.</td>
+		<td>Version 2.0</td>
+		<td>n.a.</td>
+		<td>n.a.</td>
+	</tr>
+</table>
+
+<h3>Template (v2.0)</h3>
+<h4 id="tmpl-mgrt-nt">Template migration notes</h4>
+
+<h5>Page feedback form setup code</h5>
+<div class="row">
+	<div class="col-md-6">
+		<h6>Version 2.0</h6>
+		<p>Check <a href="https://github.com/wet-boew/GCWeb/blob/master/sites/main-footer/inc-report-problem-en.html" target="_blank">this page</a> for the html inserted by the Data Ajax plugin </p>
+		<pre>
+			<code>
+				<ins>
+					&lt;div data-ajax-replace="/sites/main-footer/inc-report-problem-en.html" &gt;
+						&lt;a class="btn btn-default btn-block" href="https://www.canada.ca/en/report-problem.html"&gt;Report a problem on this page&lt;/a&gt;
+					&lt;/div&gt;
+				</ins>
+			</code>
+		</pre>
+	</div>
+	<div class="col-md-6">
+		<h6>Before version 2.0</h6>
+		<pre>
+			<code>
+				&lt;details class="brdr-0"&gt;
+					&lt;summary class="btn btn-default text-center"&gt;Report a problem on this page&lt;/summary&gt;
+					&lt;div class="well row"&gt;
+						&lt;div class="gc-rprt-prblm"&gt;
+							&lt;div class="gc-rprt-prblm-frm gc-rprt-prblm-tggl" id="wb-auto-4"&gt;
+								{ feedback form }
+							&lt;/div&gt;
+							&lt;div class="gc-rprt-prblm-thnk gc-rprt-prblm-tggl hide" id="wb-auto-5"&gt;
+								&lt;h3&gt;Thank you for your help!&lt;/h3&gt;
+								&lt;p&gt;You will not receive a reply. For enquiries, please &lt;a href="https://www.canada.ca/en/contact.html"&gt;contact us&lt;/a&gt;.&lt;/p&gt;
+							&lt;/div&gt;
+						&lt;/div&gt;
+					&lt;/div&gt;
+				&lt;/details&gt;
+			</code>
+		</pre>
+	</div>
+</div>
+
+
+<h5>Page feedback form html code</h5>
+<div class="row">
```

Comment:
```suggestion
<p>Reference implementation of the ajax files</p>
<ul>
<li><a href="../ajax/report-problem-en.html">Version 2</a></li>
<li><a href="../ajax/report-problem-aa-en.html">Version 2 with Adobe Analytic configuration</a></li>
<li><a href="../ajax/report-problem-v1-en.html">Version 1</a></li>
</ul>
<div class="row">
```
note: the version 1 is optional because it is deprecated and I do think you already documented in the documentation page.

Reply:
Changed. Thanks